pkg/agent/containerd/containerd.go

```diff
@@ -126,40 +126,52 @@ func Run(ctx context.Context, cfg *config.Node) error {
 		}
 	}
 
+	return preloadImages(cfg)
+}
+
+func preloadImages(cfg *config.Node) error {
 	fileInfo, err := os.Stat(cfg.Images)
 	if err != nil {
```

Comment:
Can you check for os.IsNotExist and just return `nil` so that it won't fail when images folder doesn't exist

Reply:
Actually nevermind, the original PR has this too.  I'll just merge and clean up.  Thanks